src/Infrastructure/Captcha/GResponseModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Runtime.Serialization;
+
+namespace FSH.WebApi.Infrastructure.Captcha;
+
+/// <summary>
+/// Google's response property naming is embarrassingly inconsistent,
+/// that's why we have to use DataContract and DataMember attributes,
+/// so we can bind the class from properties that have naming
+/// where a C# variable by that name would be against the language specifications...
+/// (i.e., '-').
+/// </summary>
+[DataContract]
```

Comment:
I don't see why that wouldn't work with System.Text.Json serialization? There's a JsonPropertyName attribute that does exactly the same like DataMember.Name.

Also... Please keep the actual property names of the class consistent with C# naming... You can put an attribute on all of them to specify the name in json.